### PR TITLE
Fix verify proto CI

### PR DESCRIPTION
**Pull request by sanchda**
# What does this PR do?

It looks like the protobuf definitions we expect from the agent have changed.  It doesn't seem that they changed in a consequential way?  This PR rectifies that without making any code-based changes.

@thedavl note:
some imports changed in the agent proto files, breaking our code generation if we copied them directly. the CI test has been modified to replace/normalize these paths.
- Side note, I expected agent proto files to change frequently, but not this frequently. Given that the agent tries to be backwards compatible anyways, maybe we don't need to keep our proto files this up to date? 

# Motivation

To make CI work lol

# Additional Notes

I didn't verify that these changes are logical.

# How to test the change?

### CODE REVIEW

**Review by morrisonlevi — DISMISSED** [review]
It looks good to me other than some funky indentation.

File: trace-protobuf/src/pb/stats.proto
[the remark this answers is not in the document]
Comment (thedavl) [reply trace-protobuf/src/pb/stats.proto:1]: looks like it's wrong in the agent proto file as well: https://raw.githubusercontent.com/DataDog/datadog-agent/main/pkg/proto/datadog/trace/stats.proto

File: trace-protobuf/src/pb/gogo.proto [hunk 1]
```diff
@@ -1,144 +0,0 @@
-// Protocol Buffers for Go with Gadgets
```
Comment (thedavl) [inline trace-protobuf/src/pb/gogo.proto:1]: previously `stats.proto` imported this. it no longer does, so we can kill it

**Review by thedavl — APPROVED** [review]
`cargo build --features generate-protobuf` re-generates `pb.rs` correctly, and CI is 🟢

**Review by thedavl — APPROVED** [review]

**Comment by sanchda** [comment]
Thank you for the fixups, @thedavl!

**Comment by sanchda** [comment]
We get a report_glitlab_CI_status failure, but it appears to be related to the runner not this PR.  Going to go ahead and merge since we're otherwise all green.

**Pull request merged**